src/SDK/panorama/IUIPanel.h

```diff
@@ -73,14 +73,14 @@ namespace panorama
         virtual IUIPanel* FindChildInLayoutFile(const char *) = 0;
         virtual void unk11() = 0;
         virtual IUIPanel* FindPanelInLayoutFile(const char *) = 0;
-        virtual bool IsDecendantOf(IUIPanel const * const) = 0;
+        virtual bool IsDecendantOf(IUIPanel * const) = 0;
```

Comment:
here it is transforming the const pointer to a const object into just a const pointer which might work but it should stay as it is.

Reply:
Parameter 1 is const qualified in the function declaration; const qualification of parameters only have effect in function definition